changelogs/unreleased/gh-6253-net-box-iproto-id.md

```diff
@@ -0,0 +1,6 @@
+## feature/core
```

Comment:
For the commit message:
- `currently, both sets are empty, but this will change in the future` - shouldn't they be non-empty already now? According to the patch they are not empty. Netbox supports both streams and transactions. As well as the version of tarantool server you are working on.
- From the names `required_protocol_features` and `peer_protocol_features` I would drop the word 'protocol'. Up to you.
- `there are two features are defined` - one 'are' is excess.

Reply:
1 and 3 fixed. As for 2, I prefer to leave 'protocol' in the names for consistency with `required_protocol_version` and `peer_protocol_version`. We can't drop 'protocol' from the latter, because otherwise one could confuse it with `peer_version_id`, which stores Tarantool version of the server.